### PR TITLE
Fixing bug with imported bundles

### LINKED ISSUE #21341: We are unable to show the complete list of contents after upload large bundles
When you push or import large bundles and then go to the publish queue, we are showing just a preview of the first 3 contents, but If you try to see the complete list we are unable to display it. 

https://user-images.githubusercontent.com/2641437/144469404-fe0428d3-1e84-48bc-b0af-14a9c44c46c7.mov

**To Reproduce**
1. Start a dotCMS instance
2. Import a large bundle (more than 10 pieces of content)
3. Go to publish queue 
4. Try to see the complete list of contents of the bundle 

**Expected behavior**
We need be able to display the whole list without problems 

**Screenshots**
image (1)

**Desktop (please complete the following information):**
 Tested on release-21.12 // Docker // FF

### PR DESCRIPTION
**Pull request by freddyucv**
The imported bundles are not saved in the publishing_bundle tables so find them by the BundleAPIImpl was not working

### CODE REVIEW

**Review by victoralfaro-dotcms — APPROVED** [review]
LGTM

**Review by nollymar — CHANGES_REQUESTED** [review]
An IT can be included to cover this case

**Review by nollymar — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Review by erickgonzalez — CHANGES_REQUESTED** [review]
Test failing
https://raw.githack.com/dotCMS/test-results/issue-21341-We-are-unable-to-show-the-complete-list-of-contents-after-upload-large-bundles/projects/core/current/curl/postgres/reports/html/Bundle_Resource.html

File: dotCMS/src/main/java/com/dotcms/rest/BundleResource.java [hunk 1]
```diff
@@ -160,7 +161,8 @@ public Response getPublishQueueElements(@PathParam("bundleId") final String bund
             if (UtilMethods.isSet(queueElements)) {
                 detailedAssets = publishQueueElementTransformer.transform(queueElements);
             } else {
-                final PublishAuditStatus publishAuditStatus = PublishAuditAPI.getInstance()
+                final int limit = UtilMethods.isSet(limitParam) ? limitParam : NO_LIMIT_ASSETS;
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/rest/BundleResource.java:164]: we have this everywhere, we should create something such as UtilMethods.isSetOrGet(limitParam, NO_LIMIT_ASSETS)

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/BundleResource.java (hunk 1 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/rest/BundleResource.java]: done https://github.com/dotCMS/core/pull/21384/commits/855c672916721e9296f4a33af272cebc419b2d62

**Comment by freddyucv** [comment]
> Test failing https://raw.githack.com/dotCMS/test-results/issue-21341-We-are-unable-to-show-the-complete-list-of-contents-after-upload-large-bundles/projects/core/current/curl/postgres/reports/html/Bundle_Resource.html

done https://github.com/dotCMS/core/pull/21384#pullrequestreview-829085881

**Pull request merged**